src/components/Trade/hooks/useSwapper/useSwapper.ts

```diff
@@ -89,58 +91,44 @@ export const useSwapper = () => {
     return swapper.getDefaultPair()
   }, [swapperManager, bestSwapperType])
 
+  const sellAssetBalance =
+    useAppSelector(state =>
+      selectPortfolioCryptoBalanceByAssetId(state, sellAsset?.currency?.caip19),
+    ) ?? '0'
```

Comment:
nit pick - but i'd probably update the selector here to return `'0'` instead of doing this in the view layer (which it should have done in the first place anyway, prob my fault)

https://github.com/shapeshift/web/blob/develop/src/state/slices/portfolioSlice/selectors.ts#L204